docs/about/licensing.md

```diff
@@ -5,31 +5,38 @@ sidebar_label: Amplication’s licensing model
 slug: /about/licensing
 ---
 
-# Amplication’s Licensing Model
+# Amplication's Licensing Model
```

Comment:
@mulygottlieb - Can you please review it as well? I'll be happy to get your thoughts here

Reply:
Ok, thanks!
@yuval-hazaz - Can you please review the updates in the licensing file?